modules/cluster-image-registry-operator.adoc

```diff
@@ -34,11 +34,11 @@ If insufficient information is available to define a complete `image-registry` r
 The Cluster Image Registry Operator runs in the `openshift-image-registry` namespace and it also manages the registry instance in that location. All configuration and workload resources for the registry reside in that namespace.
 
 ifdef::cluster-caps[]
-In order to integrate the image registry into the cluster's user authentication and authorization system, a service account token secret and an image pull secret are generated for each service account in the cluster.
+In order to integrate the image registry into the cluster's user authentication and authorization system, a dockercfg image pull secret is generated for each service account in the cluster.
 
 [IMPORTANT]
 ====
-If you disable the `ImageRegistry` capability or if you disable the integrated {product-registry} in the Cluster Image Registry Operator's configuration, the service account token secret and image pull secret are not generated for each service account.
+If you disable the `ImageRegistry` capability or if you disable the integrated {product-registry} in the Cluster Image Registry Operator's configuration, the dockercfg image pull secret is not generated for each service account.
```

Comment:
This seems redundant.

Reply:
@sanchezl What about this seems redundant?

Reply:
I was thinking relative to the previous sentence. I read it as "we do this if you want" and "warning: we also don't do it if you don't want" . Maybe they can just be combined:

In order to integrate the image registry into the cluster's user authentication and authorization system, 
when the integrated {product-registry} is enabled in the Cluster Image Registry Operator's configuration
an image pull secret is generated for each service account in the cluster.

Reply:
That's a bit too long/complicated of a sentence.

I'm open to reworking it if necessary, but each of the existing sentences are already pretty long/complicated as they are.

To me, it makes sense as is, because first we tell you what the default normal behavior is. And then, if the user does something different (disabling the image registry), we say that the image pull secret isn't created.

However, I'm open to other suggestions.